tests/test_htmIndex.py

```diff
@@ -198,6 +219,14 @@ def testIngest(self):
                 config=default_config)
         # test with multiple files and correct config
         default_config.mag_err_column_map = {'a': 'a_err', 'b': 'b_err'}
+        default_config.ra_err_name = "ra_err"
+        default_config.dec_err_name = "dec_err"
+        default_config.pm_ra_name = "pm_ra"
+        default_config.pm_dec_name = "pm_dec"
+        default_config.epoch_name = "unixtime"
+        default_config.pm_ra_err_name = "pm_ra_err"
+        default_config.pm_dec_err_name = "pm_dec_err"
+        default_config.pm_scale = 1.0e-3  # arcsec/yr --> mas/yr
```

Comment:
There's a `config` that gets created and thrown away in `setUpClass`. Why not make it an instance variable and use it here?

Reply:
I personally think this particular test is more readable by constructing the config each time, though I will change it to not re-use the same variable name for different configs.